### PR TITLE
Docs: add how to add static library

### PR DESCRIPTION
**Pull request by sunghan-chang**
When we want to include static (built) library, there are two methods.
This document shows them.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by sunghan-chang** [comment]
For markdown view, please visit [this link](https://github.com/sunghan-chang/TizenRT/blob/readme/docs/HowToAddStaticLibrary.md).

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/HowToAddStaticLibrary.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# How to add Static Library
+
+There are two methods to include static library, [adding it into arch library](#adding-it-into-arch-library) and [adding it as a new library](#adding-it-as-a-new-library).
+
+## Adding it into arch library
+
+Makefile or Make.defs on arch can includes it for architecture specific library as belows.
```
[the remark this answers is not in the document]
Comment (sunghan-chang) [reply docs/HowToAddStaticLibrary.md]: Done. Thank you for helping.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**